### DISCUSSION TITLE
Add the ability to specify a date of birth to send congratulations with a unique coupon

### DISCUSSION THREAD

**Discussion by leyakoreya** [hizzle-co/noptin#598] [Ideas]
Your solution simply requires adding the ability to wish customers a happy birthday. To do this, you need to collect birthday information from customers and automate the mailing with congratulations and a unique coupon.

Think about it and let me know

**Comment by picocodes** [comment]
This is now possible for newsletter subscribers. WooCommerce customers and WordPress users compat to be added soon.
